coderd/provisionerdserver/provisionerdserver_test.go

```diff
@@ -4031,9 +4031,16 @@ func TestServer_ExpirePrebuildsSessionToken(t *testing.T) {
 	// Given: a prebuilt workspace where an API key was previously created for the prebuilds user.
 	var (
 		ctx             = testutil.Context(t, testutil.WaitShort)
-		srv, db, ps, pd = setup(t, false, nil)
-		user            = dbgen.User(t, db, database.User{})
-		template        = dbgen.Template(t, db, database.Template{
+		srv, db, ps, pd = setup(t, false, &overrides{
+			deploymentValues: &codersdk.DeploymentValues{
+				Sessions: codersdk.SessionLifetime{
+					DefaultTokenDuration: serpent.Duration(24 * time.Hour),
+					MaximumTokenDuration: serpent.Duration(48 * time.Hour),
+				},
+			},
+		})
```

Comment:
We can just do this in `setup`
`deploymentValues := coderdtest.DeploymentValues(t)`

Not have to set overrides